histogram/prometheus_model.go

```diff
@@ -79,6 +80,32 @@ func BucketsAsJson[BC model.BucketCount](buckets []APIBucket[BC]) [][]interface{
 	return ret
 }
 
+func BucketsAsString[BC model.BucketCount](buckets []APIBucket[BC]) string {
```

Comment:
As said above, I assume you can use the `String` method of the model histograms, and then you don't need this helper.